src/views/forms/FormProfileCreation/FormProfileCreationTs.ts

```diff
@@ -155,6 +155,7 @@ export class FormProfileCreationTs extends Vue {
      * Error notification handler
      */
     private errorNotificationHandler(error: any) {
+        console.log(error.message, error.errorCode);
```

Comment:
do we need a console log here?  same below

Reply:
updated